Python-source/lambda_handler.py

```diff
@@ -0,0 +1,70 @@
+# lambda-query-geogratis
+
```

Comment:
Folder name inside the repo must be lower case. Add this folder as subfolder of cloud formation.